hedera-node/src/test/java/com/hedera/services/sigs/SigOpsRegressionTest.java

```diff
@@ -281,6 +291,74 @@ public void deniesInactiveComplexOtherPartySig() throws Throwable {
 		assertFalse(invokeOtherPartySigActivationScenario(knownSigs));
 	}
 
+	@Test
+	public void deniesScheduleCreateSigActivation() throws Throwable {
+		// given:
+		setupFor(SCHEDULE_CREATE_MISSING_ADMIN);
+		// and:
+		List<TransactionSignature> unknownSigs = PlatformSigOps.createEd25519PlatformSigsFrom(
+				platformTxn.getTxn().getScheduleCreation().getSigMap(),
+				new BodySigningSigFactory(platformTxn.getTxn().getScheduleCreation().getTransactionBody().toByteArray())
+		).getPlatformSigs();
+
+		List<TransactionSignature> knownSigs = asKind(List.of(
+				new AbstractMap.SimpleEntry<>(unknownSigs.get(0), INVALID)));
+
+		// expect:
+		assertFalse(invokeOtherPartySigActivationScenario(knownSigs));
+	}
+
+	@Test
+	public void validatesScheduleCreateSigActivation() throws Throwable {
+		// given:
+		setupFor(SCHEDULE_CREATE_MISSING_ADMIN);
+		// and:
+		List<TransactionSignature> unknownSigs = PlatformSigOps.createEd25519PlatformSigsFrom(
+				platformTxn.getTxn().getScheduleCreation().getSigMap(),
+				new BodySigningSigFactory(platformTxn.getTxn().getScheduleCreation().getTransactionBody().toByteArray())
+		).getPlatformSigs();
+
+		List<TransactionSignature> knownSigs = asKind(List.of(
+				new AbstractMap.SimpleEntry<>(unknownSigs.get(0), VALID)));
+
+		// expect:
+		assertTrue(invokeOtherPartySigActivationScenario(knownSigs));
+	}
+
+	@Test
+	public void deniesScheduleSignSigActivation() throws Throwable {
+		// given:
+		setupFor(SCHEDULE_SIGN_KNOWN_SCHEDULE);
+		// and:
+		List<TransactionSignature> unknownSigs = PlatformSigOps.createEd25519PlatformSigsFrom(
+				platformTxn.getTxn().getScheduleSign().getSigMap(),
+				new BodySigningSigFactory(SCHEDULE_TX_BODY)
+		).getPlatformSigs();
+
+		List<TransactionSignature> knownSigs = asKind(List.of(
+				new AbstractMap.SimpleEntry<>(unknownSigs.get(0), INVALID)));
+
+		// expect:
+		assertFalse(invokeScheduleSigActivationScenario(knownSigs));
+	}
+
+	@Test
+	public void validatesScheduleScheduleSigActivation() throws Throwable {
+		// given:
+		setupFor(SCHEDULE_SIGN_KNOWN_SCHEDULE);
+		// and:
+		List<TransactionSignature> unknownSigs = PlatformSigOps.createEd25519PlatformSigsFrom(
+				platformTxn.getTxn().getScheduleSign().getSigMap(),
+				new BodySigningSigFactory(SCHEDULE_TX_BODY)
+		).getPlatformSigs();
+
+		List<TransactionSignature> knownSigs = asKind(List.of(
+				new AbstractMap.SimpleEntry<>(unknownSigs.get(0), VALID)));
+
+		// expect:
+		assertTrue(invokeScheduleSigActivationScenario(knownSigs));
+	}
+
```

Comment:
I think that it will be useful to check the use case in which we use signer for the schedule create TX that is the same as one of the signers in the `sigMap` for the `scheduled tx`